src/_locales/en/messages.json

```diff
@@ -122,12 +121,11 @@
 	},
 
 	"SelectorText": { "message": "Text" },
+	"SelectorFile": { "message": "File" },
 	"ConstantValue": { "message": "Constant value" },
 	"SelectorInputValue": { "message": "Input value" },
 	"SelectorLink": { "message": "Link" },
 	"SelectorPopupLink": { "message": "Popup Link" },
-	"SelectorImage": { "message": "Image" },
```

Comment:
We dont want to remove SelectorImage and SelectorDocument. We just wanted one common feature in ui for them.